client/cjdroute2.c

```diff
@@ -730,13 +721,10 @@ int main(int argc, char** argv)
                         logger,
                         allocator);
 
-    // --------------------- noBackground ------------------------ //
-
-    int64_t* noBackground = Dict_getInt(&config, String_CONST("noBackground"));
-    if (forceNoBackground || (noBackground && *noBackground)) {
```

Comment:
If I'm reading this right, upgrade is going to cause unintended effects for people who set nobg in the config file. +1 for removing it from `--genconf` though.